### PR TITLE
feat(input-textarea): allow enabling auto resizing of textarea on supported browsers

### PR DESCRIPTION
**Pull request by brc-dd**

### CODE REVIEW

File: lib/components/SInputTextarea.vue [hunk 1]
```diff
@@ -48,6 +49,12 @@ const classes = computed(() => [
   { fill: props.rows === 'fill' }
 ])
 
+const style = computed(() => {
+  return props.autoResize
+    ? `field-sizing: content; max-height: ${props.autoResize === true ? 10 : props.autoResize}lh;`
```
Comment (brc-dd) [inline lib/components/SInputTextarea.vue:54]: can't use object style syntax because it's still a non-standard attribute

**Review by kiaking — COMMENTED** [review]
@brc-dd 
I've updated the code a bit to take `rows` into account. I want it to set `min-width`, even when auto-resize is on. For example start from 1 line, etc.

I think I managed to do it, but... what do you think? Still we shouldn't use `fill` together with auto-resize.

If this looks good, I'll update the docs as well 👀

**Comment by brc-dd** [comment]
Tested the story, seems to work fine.

**Comment by kiaking** [comment]
Thanks! And yup I meant `min-height` 😅

**Comment by kiaking** [comment]
I'll update the docs

**Review by kiaking — APPROVED** [review]

**Comment by kiaking** [comment]
@brc-dd 
Updated! Please merge this if it looks good to you 🙏

**Pull request merged**